scripts/globals/mobskills/sub-zero_smash.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+--  Sub-Zero Smash
+--
+--  Description: Additional Effect: Paralysis. Damage varies with TP.
+--  Type: Physical (blunt)
+--  Range: Cone (5' yalms)
+--  Notes: This spell should be used anytime the target is behind the mob.
+--         However the online documentation suggests that this spell can
+--         still be used anytime. As a result, any other Ruszor spells
+--         should not trigger if the target is behind the mob.
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
```

Comment:
Caught it before I could comment on it 👍

Reply:
;)